package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "opstooling-testing",
+  "version": "0.0.0",
+  "description": "Test helpers and fixtures for opstooling projects",
+  "repository": "git@github.com:paritytech/opstooling-testing.git",
+  "author": "Parity <admin@parity.io> (https://parity.io)",
+  "license": "Apache-2.0",
+  "private": false,
+  "type": "commonjs",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "dependencies": {
+    "selfsigned": "^2.1.0",
+    "mockttp": "^3.6.2",
+    "node-fetch": "^2.6.7",
+    "opstooling-js": "https://github.com/paritytech/opstooling-js#v0.0.17"
+  },
+  "devDependencies": {
+    "@octokit/webhooks-types": "^5.8.0",
+    "@types/node-fetch": "^2.6.2",
+    "@types/node": "^16",
+    "opstooling-js-style": "https://github.com/paritytech/opstooling-js-style#83439524ac5201d433f19c66b3a43977fbb801b8",
+    "ts-patch": "^2.0.2",
+    "typescript": "^4.6.2",
+    "typescript-transform-paths": "^3.4.4"
+  },
+  "scripts": {
+    "build": "tsc",
+    "build_watch": "tsc -w",
```

Comment:
Just a nitpick but it seems like the convention is to have `:` instead of `_` in package scripts.

Reply:
It's kind of an old habit of mine... Maybe today's the day to change it!